src/index.html

```diff
@@ -7,8 +7,84 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" crossorigin="/"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div> 
+    
+      <nav class="nav">
+        <ul class="navLinks">
+          <li>
+            <a 
+              class="nav_a is-active list" 
```

Comment:
```suggestion
              class="nav__link is-active" 
```
`nav__link` will be more clear name; and remove class `list` if you do not use it